src/Windows/Avalonia.Win32/WindowImpl.cs

```diff
@@ -425,15 +493,6 @@ protected virtual IntPtr CreateWindowOverride(ushort atom)
                 IntPtr.Zero);
         }
 
-        bool ShouldIgnoreTouchEmulatedMessage()
```

Comment:
Again, `git blame` and anyone who would try to revert this commit in the future would be greatly confused.